test/test_integration_list.py

```diff
@@ -159,20 +159,20 @@ def test_list_limit_long_returns_same_number_of_resources(self):
     @integration_test(True)
     def test_list_limit_invalid_param_returns_empty_list(self):
         output = self.invoke_cli(self.cli_auth_params, ['list', '-l', '0'], exit_code=1)
-        self.assertIn("500 Server Error", output)
+        self.assertIn("422 Client Error", output)
```

Comment:
why was this changed?

Reply:
Because of the change in conjur version in appliance